src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -37,6 +37,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\ArgumentNullException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\ArgumentOutOfRangeException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\ArithmeticException.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Array.cs" Condition="'$(TargetsCoreRT)' != 'true'" />
```

Comment:
What are the problematic parts of this to land in CoreRT?

Reply:
There aren't any but it's a big file to clean up and I didn't want to force the work as part of the merge

Reply:
@jkotas anything else to be done?

Reply:
I have looked at the diff with CoreRT and it looks pretty manageable. Feel free to remove the condition - I can deal with adjusting CoreRT in the mirror PR.